templates/nginx/galaxy.j2

```diff
@@ -7,6 +7,7 @@ server {
     # Our log files will go here.
     access_log  /var/log/nginx/access.log;
     error_log   /var/log/nginx/error.log;
+    sendfile    on;
```

Comment:
Do we have the _x_accel_redirect stuff already in the nginx conf?

Reply:
https://docs.galaxyproject.org/en/master/admin/nginx.html?highlight=jbrowse#sending-files-with-nginx

Reply:
And the galaxy.yml setting also

Reply:
we have this in our group_vars/galaxyservers.yml but it's commented out some other options I was not sure if we were using 
    # nginx_x_accel_redirect_base: /_x_accel_redirect

we also had the block in nginx configs on galaxy qld

location /_x_accel_redirect/ {
    internal;
    alias /;
}

Reply:
I'll add these in.  Are these instead of of in addition to using 'sendfile on'?

Reply:
Maybe, lets leave both and see what happens